.changeset/healthy-poets-compare.md

```diff
@@ -0,0 +1,11 @@
+---
+'@verdaccio/config': patch
```

Comment:
:) above says feat, maybe you want also here to be the same

Reply:
fixed - removed fixes :) , good catch